plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHivePartitionedBucketFunction.java

```diff
@@ -41,6 +41,8 @@
 import static io.trino.plugin.hive.util.HiveBucketing.BucketingVersion.BUCKETING_V1;
 import static io.trino.plugin.hive.util.HiveBucketing.BucketingVersion.BUCKETING_V2;
 import static io.trino.spi.type.BigintType.BIGINT;
+import static java.util.Collections.max;
```

Comment:
Commit message
`Add a test that buckets generated for partition are consecutive` -> `Test that buckets generated for partition are consecutive